nix/automation/pipelines.nix

```diff
@@ -0,0 +1,29 @@
+{
+  inputs,
+  cell,
+}: {
+  ci = {config, lib, ...}: {
+    preset = {
+      nix.enable = true;
+
+      github.ci = {
+        # Tullia tasks can run locally or on Cicero.
```

Comment:
how can I run it locally?
can u elaborate more, maybe some README with references? :)

Reply:
@dermetfan I'd like to second this. This seems to be "Tullia" task description. Where is the documentation for this? How would we know what configurations are available here? (Looks like a nix module system)